src/hb-subset-cff-common.hh

```diff
@@ -1038,12 +1049,25 @@ struct subr_subsetter_t
 
   }
 
+  parsed_cs_str_t& get_parsed_charstring (unsigned i)
+  {
+    if (cached_charstrings.length) return *(cached_charstrings[i]);
```

Comment:
vector has operator bool so you can just say `if (cached_charstrings)`.

Reply:
Fixed